tests/phpunit/CRM/Contact/BAO/GroupContactTest.php

```diff
@@ -329,7 +329,7 @@ public function testbyGroupType() {
     foreach ($useCases as $case) {
       $query = new CRM_Contact_BAO_Query(CRM_Contact_BAO_Query::convertFormValues($case['form_value']));
       list($select, $from, $where, $having) = $query->query();
-      $groupContacts = CRM_Core_DAO::executeQuery("SELECT DISTINCT contact_a.id $from $where ORDER BY contact_a.first_name")->fetchAll();
+      $groupContacts = CRM_Core_DAO::executeQuery("SELECT DISTINCT contact_a.id, contact_a.first_name $from $where ORDER BY contact_a.first_name")->fetchAll();
```

Comment:
just adds first_name into the select criteria, given that id should be unique anyway shouldn't make an issue

Reply:
Agree afterall its UT, and you are fixing the query in a right way :D